packages/api-gateway/src/drivers/http/plugins/services.js

```diff
@@ -1,12 +1,10 @@
-'use strict';
-
-const MathServices = require('@booking-services/math');
+const placesService = require('@booking-services/places');
 
 const PluginLoader = require('fastify-plugin');
 
 async function services(fastify) {
 
-	await fastify.decorate('mathServices', MathServices)
+	await fastify.decorate('places', placesService)
```

Comment:
@Alejandrocuartas We shouldn't remove the mathServices yet, until all teams integrate their packages.
```suggestion
	await fastify.decorate('mathServices', MathServices);
         await fastify.decorate('placesServices', placesServices)
```